fiftyone/core/odm/dataset.py

```diff
@@ -376,6 +376,8 @@ class DatasetAppConfig(EmbeddedDocument):
         media_fallback (False): whether to fall back to the default media field
             (``"filepath"``) when the alternate media field value for a sample
             is not defined
+        disable_frame_filtering (False): whether to disable frame filtering for
```

Comment:
FYI- I updated the description here to prepare for the intended state where this setting only affects the grid view